fake-snippets-api/lib/db/seed.ts

```diff
@@ -1,4 +1,11 @@
-import type { DbClient } from "./db-client"
+import { DbClient } from "./db-client"
+
+const loadAutoloadSnippets = async (db: DbClient) => {
+  if (process.env.AUTOLOAD_SNIPPETS === "true") {
```

Comment:
this function name is misleading because it says `loadAutoloadSnippets` but you're conditionally loading the snippets

i would just move the condition to the call site. But if for whatever reason you wanted this function to contain the condition it would need to be renamed to indicate that it's behavior is condition, `loadAutoloadSnippetsIfConfigured` for example (it's an uncommon way to do things though)